src/todos.js

```diff
@@ -0,0 +1,32 @@
+export default class Todos {
+  constructor() {
+    this.list = localStorage.getItem('todos')
+      ? JSON.parse(localStorage.getItem('todos'))
+      : [];
+  }
+
+  addTodo(todo) {
+    this.list.push(todo);
+    localStorage.setItem('todos', JSON.stringify(this.list));
+  }
+
+  removeTodo(todoID) {
+    console.log(todoID);
+    this.list = this.list.filter((todo) => todo.id !== todoID);
+    this.list.forEach((todo, index) => {
+      todo.index = index + 1;
+    });
+    const newTodos = this.list;
+    localStorage.setItem('todos', JSON.stringify(newTodos));
+  }
+
+  editTodo(todoId, todoDescription) {
+    const newData = this.list.map((todo) => {
+      if (todo.index === todoId) {
+        return { ...todo, description: todoDescription };
+      }
+      return todo;
+    });
+    localStorage.setItem('todos', JSON.stringify(newData));
+  }
```

Comment:
- [ ] When I try to edit a task description, it stays fine on the screen but after reloading, nothing is saved to localStorage. Kindly work on the edit functionality for the task description and get the edited values saved to LlocalStorage which is again rendered to the screen when the page reloads. 👍

![image](https://user-images.githubusercontent.com/50941074/199973893-26b7b128-7dd4-4fe1-8527-17d62d5c6ff3.png)

Reply:
- Thanks 🙏 for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊